src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -106,12 +106,17 @@ internal partial class BoundFixedLocalCollectionInitializer
 
     internal partial class BoundStackAllocArrayCreation
     {
-        protected override ImmutableArray<BoundNode> Children => ImmutableArray.Create<BoundNode>(this.Count);
+        internal static ImmutableArray<BoundExpression> GetChildInitializers(BoundArrayInitialization arrayInitializer)
```

Comment:
private?

Reply:
should be either `protected` or `internal`. we could use `private`, make `Children` sealed and remove the override from ConvertedStackAlloc, but I think that's not practiced here.